openrtb_ext/device.go

```diff
@@ -9,16 +9,6 @@ import (
 	"github.com/prebid/prebid-server/errortypes"
 )
 
-// PrebidExtKey represents the prebid extension key used in requests
-const PrebidExtKey = "prebid"
-
-// PrebidExtBidderKey represents the field name within request.imp.ext.prebid reserved for bidder params.
-const PrebidExtBidderKey = "bidder"
-
-const OriginalBidCpmKey = "origbidcpm"
```

Comment:
Sorry, I may have confused you here but my earlier suggestion was just to move `OriginalBidCpmKey` and `OriginalBidCurKey` to `bid.go`. The other two don't seem like they belong here either but I don't think they belong in `bid.go` too. They seem to be used more from the reference of an imp ext, from a quick look. So may be `openrtb_ext/imp.go` is a better place for them? But anyhow, we can just leave them as is for this PR and just make sure the new constants you're adding are in the right place.